joi-validations/chat.validations.js

```diff
@@ -0,0 +1,24 @@
+const {uuid, array, object, string} = require('./general.validations');
+
+const ChatValidation = {
+  sendFollowSystemMessage: {
+    body: {
+      channel_id: uuid.required(),
+      target_follow_user_id: uuid.required()
+    }
+  },
+  findOrCreateChannelBySignedSender: {
+    members: array(uuid).min(1)
+  },
+  moveToAnon: {
+    body: object({
+      anon_user_info_color_code: string.required(),
+      anon_user_info_color_name: string.required(),
+      anon_user_info_emoji_code: string.required(),
+      anon_user_info_emoji_name: string.required(),
+      targetUser: uuid.required()
+    })
+  }
+};
+
+module.exports = {ChatValidation};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The validation schema for `moveToAnon` seems to be missing some validations. It's good practice to validate the length of strings and also check if they are not empty. This can prevent potential issues with storing and processing these values later on.
```diff
  moveToAnon: {
    body: object({
      anon_user_info_color_code: string.required().min(1),
      anon_user_info_color_name: string.required().min(1),
      anon_user_info_emoji_code: string.required().min(1),
      anon_user_info_emoji_name: string.required().min(1),
      targetUser: uuid.required()
    })
  }
```
Also, it would be beneficial to add a `.max()` method to limit the maximum length of the strings. This will help to avoid any potential issues related to database storage limits or performance degradation due to handling very large strings.


<!-- This is an auto-generated comment by OSS CodeRabbit -->